src/passkey_child/passkey_child.c

```diff
@@ -93,6 +93,12 @@ int main(int argc, const char *argv[])
             ERROR("Verification error.\n");
             goto done;
         }
+    } else if (data.action == ACTION_PREFLIGHT) {
+        ret = preflight(&data, 1);
```

Comment:
1 second timeout seems low?

Reply:
Yes, the reason is explained in my previous comment